moveit_ros/perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -56,127 +57,132 @@ OccupancyMapMonitor::OccupancyMapMonitor(const std::shared_ptr<tf2_ros::Buffer>&
+  //         }
+  //
+  //         if (!sensor_list[i].hasMember("sensor_plugin"))
+  //         {
+  //           RCLCPP_ERROR(logger_occupancy_map_monitor,"No sensor plugin specified for octomap updater %d; ignoring.", i);
+  //           continue;
+  //         }
+  //
+  //         std::string sensor_plugin = std::string(sensor_list[i]["sensor_plugin"]);
+  //         if (sensor_plugin.empty() || sensor_plugin[0] == '~')
+  //         {
+  //           RCLCPP_INFO(logger_occupancy_map_monitor,"Skipping octomap updater plugin '%s'", sensor_plugin.c_str());
+  //           continue;
+  //         }
+  //
+  //         if (!updater_plugin_loader_)
+  //         {
+  //           try
+  //           {
+  //             updater_plugin_loader_.reset(new pluginlib::ClassLoader<OccupancyMapUpdater>(
+  //                 "moveit_ros_perception", "occupancy_map_monitor::OccupancyMapUpdater"));
+  //           }
+  //           catch (pluginlib::PluginlibException& ex)
+  //           {
+  //             ROS_ERROR(logger_occupancy_map_monitor,"Exception while creating octomap updater plugin loader %s ", ex.what());
+  //           }
+  //         }
+  //
+  //         OccupancyMapUpdaterPtr up;
+  //         try
+  //         {
+  //           up = updater_plugin_loader_->createUniqueInstance(sensor_plugin);
+  //           up->setMonitor(this);
+  //         }
+  //         catch (pluginlib::PluginlibException& ex)
+  //         {
+  //           RCLCPP_ERROR(logger_occupancy_map_monitor,"Exception while loading octomap updater '%s': %s",sensor_plugin, ex.what());
+  //         }
+  //         if (up)
+  //         {
+  //           /* pass the params struct directly in to the updater */
+  //           if (!up->setParams(sensor_list[i]))
+  //           {
+  //             RCLCPP_ERROR(logger_occupancy_map_monitor,"Failed to configure updater of type %s", up->getType().c_str());
+  //             continue;
+  //           }
+  //
+  //           if (!up->initialize())
+  //           {
+  //             RCLCPP_ERROR(logger_occupancy_map_monitor,"Unable to initialize map updater of type %s (plugin %s)", up->getType().c_str(),
+  //                       sensor_plugin.c_str());
+  //             continue;
+  //           }
+  //
+  //           addUpdater(up);
+  //         }
+  //       }
+  //     else
+  //       RCLCPP_ERROR(logger_occupancy_map_monitor,"List of sensors must be an array!");
+  //   }
+  //   catch (XmlRpc::XmlRpcException& ex)
+  //   {
+  //     RCLCPP_ERROR(logger_occupancy_map_monitor,"XmlRpc Exception: %s", ex.getMessage().c_str());
+  //   }
+  // }
 
   /* advertise a service for loading octomaps from disk */
-  save_map_srv_ = nh_.advertiseService("save_map", &OccupancyMapMonitor::saveMapCallback, this);
-  load_map_srv_ = nh_.advertiseService("load_map", &OccupancyMapMonitor::loadMapCallback, this);
+  // save_map_srv_ = nh_.advertiseService("save_map", &OccupancyMapMonitor::saveMapCallback, this);
```

Comment:
Why are you not porting the service? Why are you not porting the service? Do you need some help with this?

Reply:
Changes made on https://github.com/AcutronicRobotics/moveit2/commit/b04f8b30ea4153be08ce5b53abfb05296dc7442f#diff-ff2eb2808beed252d2672fd10de1b05e

Reply:
https://github.com/AcutronicRobotics/moveit2/pull/26/files/65ca84acd7b298680785bade43ed81a45657916a#r264123531

Reply:
https://github.com/AcutronicRobotics/moveit2/commit/b04f8b30ea4153be08ce5b53abfb05296dc7442f